components/ProductView/Product.tsx

```diff
@@ -14,14 +17,33 @@ import { ProductHistoryGraph } from "./ProductHistoryGraph"
 
 const locale = "ru"
 
-const ProductImage: React.FC<{ url: string }> = ({ url }) => {
+const ProductImage: React.FC<{ url: string; cls?: string }> = ({ url, cls }) => {
   return (
-    <div className="h-48 w-64 mx-auto flex justify-center items-center">
+    <div className={cx("h-48 w-64 mx-auto flex justify-center items-center", cls)}>
       {/* eslint-disable-next-line @next/next/no-img-element */}
       <img className="max-w-full max-h-full" src={url} alt="Photo" />
     </div>
   )
 }
+
+const ProductImageCarousel: React.FC<{ urls: string[] }> = ({ urls }) => {
+  return (
+    <Carousel
+      wrapAround
+      autoplay
+      autoplayInterval={5000}
+      pauseOnHover
+      defaultControlsConfig={{
+        pagingDotsClassName: "mx-1",
+      }}
+      renderCenterLeftControls={({ previousSlide }) => <ArrowBackIosIcon onClick={previousSlide} />}
+      renderCenterRightControls={({ nextSlide }) => <ArrowForwardIosIcon onClick={nextSlide} />}
+    >
+      {urls.map((url) => <ProductImage url={url} key={url} cls={"h-[31vh] pb-[35px]"} />)}
```

Comment:
Why is cls needed?

I think there should be a way to set the height of the carousel itself. And what about paddings? And also why `cls` instead of `className` like in react itself?